Documentation improvement for creating your own beat

**Describe the enhancement:**
docs/devguide/newbeat.asciidoc
Improve documentation around creating custom beats. Specifically, using `make setup` as described in [our documentation here](https://www.elastic.co/guide/en/beats/devguide/current/setting-up-beat.html) [(relevant Github file)](https://github.com/elastic/beats/edit/master/docs/devguide/newbeat.asciidoc) causes an error:

```
pi@pitest:~/go/src/github.com/robb3rt/stonksbeat $ make setup
make: *** No rule to make target 'setup'.  Stop.
```

Following through the documentation it seems that the configuration files have already been created, so perhaps this step is no longer necessary? I don't know for sure by following the documentation alone.

It would be nice if the document can clarify if this is expected or to update this document in case anything has changed in the process. It would also be nice if the document can go more into detail about what the template files are? I don't see that being linked to any examples, unlike the configuration files are.

--- a/CHANGELOG.next.asciidoc
+++ b/CHANGELOG.next.asciidoc
@@ -239,6 +239,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix ILM alias not being created if `setup.ilm.check_exists: false` and `setup.ilm.overwrite: true` has been configured. {pull}24480[24480]
 - Fix issue discovering docker containers and metadata after reconnections {pull}24318[24318]
 - Allow cgroup self-monitoring to see alternate `hostfs` paths {pull}24334[24334]
+- Fix 'make setup' instructions for a new beat {pull}24944[24944]
 
 *Auditbeat*
 

--- a/docs/devguide/newbeat.asciidoc
+++ b/docs/devguide/newbeat.asciidoc
@@ -138,7 +138,7 @@ For the `github_name`, enter your github id. The `beat` and `beat_path` are set 
 ---------
 Enter a project name [examplebeat]:  Countbeat
 Enter a github name [your-github-name]: {username}
-Enter a beat path [github.com/{username}/countbeat]: 
+Enter a beat path [github.com/{username}/countbeat]:
 Enter a full name [Firstname Lastname]: {Full Name}
 Enter the github.com/elastic/beats revision [master]:
 ---------

--- a/docs/devguide/newbeat.asciidoc
+++ b/docs/devguide/newbeat.asciidoc
@@ -162,7 +162,7 @@ To fetch dependencies and set up the Beat, run:
 [source,shell]
 ---------
 cd ${GOPATH}/src/github.com/{user}/countbeat
-make setup
+make update
 ---------
 
 The Beat now contains the basic config file, `countbeat.yml`, and template files. The Beat is "complete" in the sense
@@ -340,7 +340,7 @@ countbeat:
 
  - `period`: Defines how often to send out events
 
-The config file is generated when you run `make setup` to <<setting-up-beat,set up the beat>>. The file contains
+The config file is generated when you run `make update` to <<setting-up-beat,set up the beat>>. The file contains
 basic configuration information. To add configuration options to your Beat, you need to update the Go structures in
 `config/config.go` and add the corresponding config options to `_meta/beat.yml`.
 

--- a/generator/_templates/beat/{beat}/README.md
+++ b/generator/_templates/beat/{beat}/README.md
@@ -16,7 +16,7 @@ To get running with {Beat} and also install the
 dependencies, run the following command:
 
 ```
-make setup
+make update
 ```
 
 It will create a clean git history for each major step. Note that you can always rewrite the history if you wish before pushing your changes.

--- a/generator/_templates/metricbeat/{beat}/README.md
+++ b/generator/_templates/metricbeat/{beat}/README.md
@@ -8,7 +8,7 @@
 To get started run the following command. This command should only be run once.
 
 ```
-make setup
+make update
 ```
 
 It will ask you for the module and metricset name. Insert the name accordingly.

--- a/x-pack/osquerybeat/README.md
+++ b/x-pack/osquerybeat/README.md
@@ -16,7 +16,7 @@ To get running with {Beat} and also install the
 dependencies, run the following command:
 
 ```
-make setup
+make update
 ```
 
 It will create a clean git history for each major step. Note that you can always rewrite the history if you wish before pushing your changes.

